includes/settings/js/src/components/fields/Form.jsx

```diff
@@ -139,6 +140,32 @@ function Form({ id, position, type, editing, storedData, hasDirtyField }) {
 				},
 			},
 		},
+		email: {
+			component: EmailSettings,
+			fields: {
+				allowedDomains: {
+					setValueAs: (value) =>
+						value ? value.replace(/\s/g, "") : "",
+					validate: {
+						formattedCorrectly: (value) => {
+							if (!value) {
+								return true;
+							}
+
+							const domains = value
+								.split(",")
+								.map((domain) => domain.trim());
```

Comment:
Is it worth stripping empty strings here so that lists like `example.net, example.com,` (trailing comma at end of list) and `example.com, ,test.example.com` (double comma) pass validation?

```suggestion
							const domains = value
								.split(",")
								.map((domain) => domain.trim())
								.filter((domain) => domain.length > 0);
```

Reply:
Nice! Will change.